mousebender/simple.py

```diff
@@ -42,6 +67,18 @@ def _normalize_project_url(url):
     return create_project_url(base_url, project_name)
 
 
+def _check_version(tag, attrs):
+    """Check if the tag is a PEP 629 tag and is a version that is supported."""
+    if tag != "meta" or attrs.get("name") != "pypi:repository-version":
+        return
+
+    major, minor = map(int, attrs["content"].split("."))
+    if major > _SUPPORTED_VERSION[0]:
```

Comment:
Totally realize that the _expectation_ here is that the major number != expected, and since it's a version number mismatch, it is _likely_ going to be higher than the expected version.

However, wouldn't just inequality (`major != _SUPPORTED_VERSION[0]`) be the most appropriate here?

Reply:
Not sure about it being the _most_ appropriate, but it would work. Since the first version is 1.0 it  inherently is accurate either way. 😄

The PEP also acknowledges the chances of the major number going passed `1` is pretty slim.